hack/generator/pkg/astmodel/testdata/NastyTest-Direct.golden

```diff
@@ -21,13 +21,10 @@ func (person Person) ConvertFrom(source *vHub.Person) error {
 			for nasty2Key, nasty2Value := range nasty1Item {
 				nasty2Map[nasty2Key] = nasty2Value
 			}
-
```

Comment:
Yeah, whitespace is for losers!
(Just kidding - I think removing these is a good thing.)